### ISSUE TITLE
Minor export/typing issue in `3.3.0-1` pre-release?

### ISSUE THREAD

**Issue by ssciolla** [KTH/canvas-api#29]
Hi `@kth/canvas-api` team! Let me preface this issue by saying my team at the University of Michigan has been using your library, and has thus far found it quite useful. I hope you see me filing this issue more as a contribution than a complaint.

I've been working on some dependency updates, and I saw in a few of your pre-releases that you are making the switch to TypeScript, which is great for us since the project we're utilizing this library in is all TypeScript. However, when trying to adopt `3.3.0-1`, I encountered an issue where I couldn't use `error instanceof CanvasApiError` (it returned an error saying "Right-hand side of 'instanceof' is not an object"), which is important to our use case. Note that I did not have this problem with the `4.0.0-0` pre-release, so I looked at the difference to see what might have changed: https://github.com/KTH/canvas-api/compare/v4.0.0-0...v3.3.0-1 . I believe the `module.exports = CanvasApi` at the bottom may be the culprit, but I am not an expert on exports/modules in JS/TS.

I saw that your test is passing when `CanvasApiError` is imported from `./utils` (it also works for me in our project if we import from `@kth/canvas-api/dist/utils`), but I think the intention was for this to be available from the root (since it's re-exported in `index.ts`). If you modify your test to import from `./index.ts`, you can reproduce the problem.

```
+++ b/src/errorHandler.test.ts
@@ -1,6 +1,5 @@
 import createTestServer from "create-test-server";
-import Canvas from "./index";
-import { CanvasApiError } from "./utils";
+import Canvas, { CanvasApiError } from "./index";

 test("errorHandler converts HTTPError to CanvasApiError", async () => {
   const server = await createTestServer();
@@ -13,6 +12,8 @@ test("errorHandler converts HTTPError to CanvasApiError", async () => {
   try {
     await canvas.get("something");
   } catch (err) {
+    console.log(err.constructor.name);
+    console.log(typeof CanvasApiError);
     expect(err).toBeInstanceOf(CanvasApiError);
     expect(err).toMatchInlineSnapshot(
       `[CanvasApiError: Response code 401 (Unauthorized)]`
(END)
```

If you're open to contributions, I can open a PR against `ts-compat` with this commit: https://github.com/ssciolla/canvas-api/commit/7f5c0e7df925565285f1ef13d61869d69023401c

Anyways, hope that is helpful. Do you have plans to release `4.0.0` (not a pre-release) in the near future? Thanks for sharing this project!

**Comment by me97esn** [comment]
Hi, glad to hear from you!
We would very much like PR's from people outside of KTH. We believe that open source and contribution is valuable for everyone.
I can't promise any timeline on when we review, merge or close the PR of course, but we'll do our best.

**Comment by exacs** [comment]
Hi @ssciolla! Thanks for reporting this problem!

Our plans are to release `4.0.0` as soon as possible. Actually `3.3.x` will be deprecated so my recommendation is that you use `4.0.0-1` (we have released it now **without** the `module.exports` as your commit proposes), specially if it works for you! It is a pre-release but we will not make breaking changes before `4.0.0`, only bug fixes and add partial retro-compatibility.

**Comment by ssciolla** [comment]
Thanks for the replies! @exacs, it sounds like you released `4.0.0-1`, but I didn't see that on `npm` (https://www.npmjs.com/package/@kth/canvas-api), though actually I do see the tag when I search for "kth/canvas-api" and see your library in the results. I see the commits on the `ts` branch though. We're currently planning to use `4.0.0-0` which works for us.

Looking forward to the `4.0.0` full release, and if there's any way I think I can contribute, I'll file a new issue/PR. Cheers!

**Comment by ssciolla** [comment]
Ah, `4.0.0-1` is there now, maybe `npm` takes a bit to fully publish a new version. Thanks!
